.github/ISSUE_TEMPLATE/dep-golang.md

```diff
@@ -74,6 +74,12 @@ SIG Release Slack thread:
 <!-- Example: https://github.com/kubernetes/test-infra/pull/20038 -->
 - [ ] `kubekins`/`krte` image variants update: 
 
+
+#### After all cherry picks are merged for kubernetes/kubernetes
+
+<!-- Example: https://github.com/kubernetes/kubernetes/pull/104226 -->
+- [ ] `staging/publishing` bot rules: 
+
```

Comment:
Let's be a little more detailed about this section here, since this will not always be required.

Something along the lines of:

```suggestion
<!--
If the Golang version of the active development branch (`master`) is newer than
any of the Golang versions on _active_ release branches, then the current
Golang versions for all release branches need to be updated within publishing
bot rules.

Concrete example:
- `master` was just updated from go1.16.6 to go1.16.7
- cherry picks were issued to the 1.22 and 1.21 branches
- `release-1.20` was also updated from go1.15.14 to go1.15.15
- these changes were cherry picked to the 1.19 branch

In this case, because we updated the default go version on `master` to
go1.16.7, there's no action required for staging repositories using go1.16.

However, for staging repository branches using go1.15, the `master` branch's 
publishing bot rules need to be updated to learn about the Golang update that
happened for the 1.20 and 1.19 Kubernetes release branches.

PR: https://github.com/kubernetes/kubernetes/pull/104226
-->
- [ ] publishing bot rule updates for active Golang versions: 

```

Reply:
<3